### PR TITLE
chore: update release

### PR DESCRIPTION
**Pull request by fzipi**
Update release.

- update modsecurity to v3.0.9
- update apache to v2.4.57
- fix README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -275,7 +275,7 @@ All these variables impact in configuration directives in the modsecurity engine
 | ALLOWED_REQUEST_CONTENT_TYPE_CHARSET | A string indicating the allowed_request_content_type_charset (Default: `utf-8\|iso-8859-1\|iso-8859-15\|windows-1252`) |
 | ALLOWED_HTTP_VERSIONS | A string indicating the allowed_http_versions (Default: `HTTP/1.0 HTTP/1.1 HTTP/2 HTTP/2.0`) |
 | RESTRICTED_EXTENSIONS | A string indicating the restricted_extensions (Default: `.asa/ .asax/ .ascx/ .axd/ .backup/ .bak/ .bat/ .cdx/ .cer/ .cfg/ .cmd/ .com/ .config/ .conf/ .cs/ .csproj/ .csr/ .dat/ .db/ .dbf/ .dll/ .dos/ .htr/ .htw/ .ida/ .idc/ .idq/ .inc/ .ini/ .key/ .licx/ .lnk/ .log/ .mdb/ .old/ .pass/ .pdb/ .pol/ .printer/ .pwd/ .resources/ .resx/ .sql/ .sys/ .vb/ .vbs/ .vbproj/ .vsdisco/ .webinfo/ .xsd/ .xsx/`) |
-| RESTRICTED_HEADERS | A string indicating the restricted_headers (Default: `/proxy/ /lock-token/ /content-range/ /if/`) |
+| RESTRICTED_HEADERS | A string indicating the restricted_headers (Default: `/accept-charset/ /content-encoding/ /proxy/ /lock-token/ /content-range/ /if/`) |
```
Comment (theseion) [inline README.md:278]: Should `/x-http-method-override/ /x-http-method/ /x-method-override/` also be included? They are in the example file.

**Review by theseion — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fzipi) [reply README.md:278]: If they are, then they should.

Comment (theseion) [reply README.md:278]: @kilbiller

Comment (kilbiller) [reply README.md:278]: They are not set in `owasp/modsecurity-crs:3.3.4-nginx-alpine-202303180303` at least. I think they were added in v4 ?

Comment (fzipi) [reply README.md:278]: Good catch. Maybe we are adding headers from v4 that should not be here 🤔

Comment (theseion) [inline README.md:278]: Remove this change. It should not be part of this PR. Rebase instead.

**Review by theseion — CHANGES_REQUESTED** [review]

**Review by theseion — APPROVED** [review]
